packages/@dcl/ecs/src/components/manual/Transform.ts

```diff
@@ -135,10 +135,10 @@ export function defineTransformComponent(
   return {
     ...transformDef,
     create(entity: Entity, val?: TransformTypeWithOptionals) {
-      return transformDef.create(entity, val as TransformType)
+      return transformDef.create(entity, { ...TransformSchema.create(), ...val } as TransformType)
```

Comment:
```suggestion
      return transformDef.create(entity, { ...TransformSchema.create(), ...val })
```
do we still need the `as` ? 
TransformSchema.create() already returns `TransformType`

Reply:
also why dont use the `TransformSchema.extend` that already does the spread operator

Reply:
good catch, fixed 👌